snapshots/cardano-1.28.0.yaml

```diff
@@ -0,0 +1,175 @@
+name: cardano-1.28.0
+
+resolver: lts-17.6
```

Comment:
```suggestion
resolver: lts-17.15
```
Can we use the latest snapshot in the LTS-17 series?

Reply:
_possibly_